lib/ui/widgets/grade_evaluation_tile.dart

```diff
@@ -78,20 +83,23 @@ class _GradeEvaluationTileState extends State<GradeEvaluationTile>
                 alignment: Alignment.topCenter,
                 child: LayoutBuilder(
                   builder: (context, constraints) {
-                    return GradeCircularProgress(
-                      constraints.maxHeight / 100,
-                      completed: widget.completed,
-                      key: Key(
-                          "GradeCircularProgress_${widget.evaluation.title}"),
-                      studentGrade: Utils.getGradeInPercentage(
-                        widget.evaluation.mark,
-                        widget.evaluation.correctedEvaluationOutOfFormatted,
-                      ),
-                      averageGrade: Utils.getGradeInPercentage(
-                        widget.evaluation.passMark,
-                        widget.evaluation.correctedEvaluationOutOfFormatted,
-                      ),
-                    );
+                    return _buildDiscoveryFeatureDescriptionWidget(
```

Comment:
Am I missing something or will this build a discovery item for every evaluation tile? Don't know if it's maybe difficult to put it on one only? (maybe with a parameter you passed to the widget when it is created) 😄